app/routes/application.vue

```diff
@@ -0,0 +1,61 @@
+<template lang="html">
+  <div class="">
+    <nav class="panel">
+      <p class="panel-heading">
+        Sign Up For My Web App
+      </p>
+
+      <p class="control">
+      <input class="input" type="text" placeholder="First Name">
+        </p>
+
+        <p class="control">
+        <input class="input" type="text" placeholder="Last Name">
+          </p>
+
+        <p class="control">
+        <input class="input" type="text" placeholder="Email Address">
+          </p>
+
+        <p class="control">
+        <input class="input" type="text" placeholder="Current website url">
+          </p>
+
+        <p class="control">
+            <span class="select">
+              <select>
+            <option>English</option>
+            <option>Spanish</option>
+              </select>
+            </span>
+        </p>
+
+      <p class="control">
+      <textarea class="textarea" placeholder="Your Comment"></textarea>
+      </p>
+
+      <p class="control">
+      <input class="input" type="text" placeholder="Mobile Number">
+        </p>
+
+      <p class="control">
+      <input class="input" type="text" placeholder="Phone Number">
+        </p>
+
+     <button class="button is-fullwidth">Submit</button>
```

Comment:
Add a `panel-block` around just this button and it will make nice padding around the button.